File: src/main/java/Library/InitializeAccounts.java
package Library;

// InitializeAccounts: Handing the initialization of accounts
public class InitializeAccounts {

    // Initializing
    public Accounts initializeAccounts(){
        Accounts accounts = new Accounts();
        accounts.add(new Borrower("alice", "pass123"));
        accounts.add(new Borrower("bob", "pass456"));
        accounts.add(new Borrower("charlie", "pass789"));
        return accounts;
    }

}

File: A3/utils/InitializeAccounts.js
const Accounts = require("../models/Accounts");
const Borrower = require("../models/Borrower");

// EXACT 1:1 conversion from Java
class InitializeAccounts {
    initializeAccounts() {
        const accounts = new Accounts();
        accounts.add(new Borrower("alice", "pass123"));
        accounts.add(new Borrower("bob", "pass456"));
        accounts.add(new Borrower("charlie", "pass789"));
        return accounts;
    }
}

module.exports = InitializeAccounts;
